maas/utils.go

```diff
@@ -109,6 +109,7 @@ func setTerraformState(d *schema.ResourceData, tfState map[string]interface{}) e
 		delete(tfState, "id")
 	}
 	for k, v := range tfState {
+		//lintignore:R001
```

Comment:
Could you please add an explanation here? Possibly a link to the check as well.

Reply:
Fixed